tests/cases.py

```diff
@@ -47,7 +45,7 @@ def test_dataset(self):
         # Load
         try:
             self.dataset._load()
-        except (ConnectionError, EOFError):
+        except (EOFError, IOError):
```

Comment:
I'm aware this change is not relevant for the PR, but the data source for FB15k is acting up and this generalizes the test to better handle it